### PR TITLE
Fix misc

### PR DESCRIPTION
**Pull request by ilalande**

### CODE REVIEW

**Review by ThomasDCW — DISMISSED** [review]
ok for me

File: frontend/src/components/SignUp/index.jsx [hunk 1]
```diff
@@ -86,30 +86,14 @@ export default function SignUp() {
...
-        toast.success(`Félicitations, vous êtes bien inscrit à WIM`, {
-          position: "bottom-center",
-          autoClose: 5000,
-          hideProgressBar: false,
-          closeOnClick: true,
-          pauseOnHover: true,
-          draggable: true,
-          progress: undefined,
-        });
+        toast.success(`Félicitations, vous êtes bien inscrit à WIM`, {});
```
Comment (LuigiMenez) [inline frontend/src/components/SignUp/index.jsx:89]: We forgot to remove the braces when we removed the options, right?

File: frontend/src/components/SignUp/index.jsx [hunk 2]
```diff
@@ -86,30 +86,14 @@ export default function SignUp() {
...
-        toast.error(`${e.response.data}`, {
-          position: "bottom-center",
-          autoClose: 5000,
-          hideProgressBar: false,
-          closeOnClick: true,
-          pauseOnHover: true,
-          draggable: true,
-          progress: undefined,
-        });
+        toast.error(`${e.response.data}`, {});
```
Comment (LuigiMenez) [inline frontend/src/components/SignUp/index.jsx:96]: Same

**Review by LuigiMenez — APPROVED** [review]
Ok for me.

**Review by LuigiMenez — CHANGES_REQUESTED** [review]
Fix Minor

**Review by ThomasDCW — APPROVED** [review]

**Review by LuigiMenez — APPROVED** [review]
Ok for me.

File: frontend/src/components/InfoSchool/index.jsx [hunk 3]
```diff
@@ -25,6 +26,11 @@ export default function InfoSchool({ iduser }) {
       })
       .then(({ data }) => {
         setFormData(data);
+        toast.success("Vos informations ont bien été modifiées");
+        setActive(" ");
```
Comment (LoicBrassart) [inline frontend/src/components/InfoSchool/index.jsx:30]: ?

**Review by LoicBrassart — APPROVED** [review]
SnE

**Pull request merged**